src/ingest_validation_tools/upload.py

```diff
@@ -627,6 +632,7 @@ def _check_url(
                     *get_entity_type_vals(response.json()),
                 )
         elif field in ["orcid_id", "orcid"]:
+            url = urljoin(constrained_fields[field], f"?q=orcid:{value}")
             headers = {"Accept": "application/json"}
             response = requests.get(url, headers=headers)
```

Comment:
Same suggestion as above

Reply:
The params for the ORCID API are so hateful, that {"q": "orcid:{value}"} pattern threw me until literally right this second. Thank you for helping me give this a second look :)